alcli/alertlogic_cli.py

```diff
@@ -249,13 +249,24 @@ def _encode(self, operation, param_name, param_value):
             p = urlparse(param_value)
             if p.scheme == "file":
                 value_file_path = os.path.abspath(os.path.join(p.netloc, p.path))
-                with open (value_file_path, "r") as value_file:
+                with open(value_file_path, "r") as value_file:
                     param_value = value_file.read()
 
         schema = operation.get_schema()
         parameter = schema[OpenAPIKeyWord.PARAMETERS][param_name]
+        type = parameter.get(OpenAPIKeyWord.TYPE)
```

Comment:
Let's pick a different name for this, e.g. `parameter_type` to avoid confusing this even more.

```
$ python3
Python 3.8.6 (default, Oct 20 2020, 10:08:37)
[Clang 11.0.3 (clang-1103.0.32.62)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> type(type)
<class 'type'>
```

Reply:
sure thing

Reply:
![0514f37913962f4884950fecdd7f8ecfd53296-wm](https://user-images.githubusercontent.com/1903630/100129311-04062780-2e47-11eb-87c7-c3caa2a22c82.jpg)